src/SearchWrapper.cpp

```diff
@@ -490,16 +490,29 @@ searchWrapper(Rcpp::CharacterVector & dbpath, Rcpp::List & enquireList, Rcpp::Li
     Xapian::Database db(path);
 
     // Create a Xapian::Query from user specified data
-    Xapian::Query query = parseQuery(queryList);
-
+    Xapian::Query query;
+    try{
+    query = parseQuery(queryList);
```

Comment:
This line needs indenting more (also spaces before `{` and after `}` tend to be more readable).